wvd-templates/DSC/Configuration.ps1

```diff
@@ -41,7 +41,10 @@ configuration CreateHostPoolAndRegisterSessionHost
         [string]$DefaultDesktopUsers,
 
         [Parameter(mandatory = $false)]
-        [string]$RDPSModSource = 'attached'
+        [string]$RDPSModSource = 'attached',
+
+        [Parameter(mandatory = $false)]
+        [bool]$EnableVerboseMsiLogging = $false

```

Comment:
I still need to test that passing a bool from the ARM template works

Reply:
I think it does, I remember trying it when I was integrating installing RDS powershell module from the gallery. I think switch didn't work well, but bool should work, but I would double check that. If possible we should update isServicePrincipal and EnablePersistentDesktop to use bool as well

Reply:
I tested this and it works with the [bool] parameter in Configuration.ps1.
I don't think we should change the isServicePrincipal and EnablePersistentDesktop  
parameters in from [string] to [bool] in  Configuration.ps1 since the terraform code [like this file that references isServicePrincipal](https://github.com/Azure/RDS-Templates/blob/master/wvd-sh/terraform-azurerm-windowsvirtualdesktop/virtual_machine_extensions.tf) would probably have to be updated and since we are going into more of a maintenance only mode.